src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,42 @@
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+
```

Comment:
The calednar should be centered both horizontally and vertically. Adding flex to body will solve this problem ;)